ehitchhiking-ui/src/app/shared/components/active-trip-additional-info/active-trip-additional-info.component.html

```diff
@@ -0,0 +1,64 @@
+<div >
+<div class  = "for-passenger" *ngIf="trip.role === 2">
```

Comment:
absolutely unclear what is 2. Pls use UserState enum here

Reply:
fixed